configs/clip.yml

```diff
@@ -0,0 +1,19 @@
+# Model
+feature_type: 'CLIP-ViT-B-32'
```

Comment:
It's a nice idea!
But the output path and the model name may not be compatible because of slash `/` (when using `on_extraction=save_numpy`). I'm sure we can handle it. It's just a reminder XD

Reply:
Yes, a nice catch! I feel that on Windows it could potentially be a problem as well. Do you think `str.replace('/', '_')` just before saving would do?

Reply:
Yes, it solves the problem.